### PR TITLE
Create mysql_password_expiry.md

### PR DESCRIPTION
**Pull request by rnovikovP**
https://docs.google.com/document/d/1Y6etfwDg3F50HB2LsCgPh-StYE0CkJhLKfmCrDWLOfU/edit#

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]

File: docs/advisors/checks/mysql_password_expiry.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# MySQL expired passwords
+
+## Description
+
+MySQL provides password-expiration capability, which enables database administrators to require that users reset their password.  Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,” in which the server permits the client to perform only those operations necessary to reset the expired password.
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_password_expiry.md:5]:
```suggestion
MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,”. In this mode, the server allows the client to perform only operations necessary for resetting the expired password.
```

File: docs/advisors/checks/mysql_password_expiry.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Such scenarios will impact the application users and this check helps prevent it.
+
+
+## Resolution
+
+To resolve this issue, one must change the user account password with [ALTER USER](https://dev.mysql.com/doc/refman/8.0/en/alter-user.html) or [SET PASSWORD](https://dev.mysql.com/doc/refman/8.0/en/set-password.html) command.
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_password_expiry.md:11]:
```suggestion
Change the user account password with [ALTER USER](https://dev.mysql.com/doc/refman/8.0/en/alter-user.html) or [SET PASSWORD](https://dev.mysql.com/doc/refman/8.0/en/set-password.html) command. 
```

File: docs/advisors/checks/mysql_password_expiry.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+## Description
+
+MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,”. In this mode, the server allows the client to perform only operations necessary for resetting the expired password.
+Such scenarios impact the application users and this check helps prevent them.
+
+
+## Resolution
+
+Change the user account password with [ALTER USER](https://dev.mysql.com/doc/refman/8.0/en/alter-user.html) or [SET PASSWORD](https://dev.mysql.com/doc/refman/8.0/en/set-password.html) command. 
+This restores normal access for the session, as well as for subsequent connections that use the account.
```
Comment (patrickbirch) [inline docs/advisors/checks/mysql_password_expiry.md:12]: This action

File: docs/advisors/checks/mysql_password_expiry.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# MySQL expired passwords
+
+## Description
+
+MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,”. In this mode, the server allows the client to perform only operations necessary for resetting the expired password.
```
Comment (rnovikovP) [inline docs/advisors/checks/mysql_password_expiry.md:5]:
```suggestion
MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. When the password expires, the server either disconnects the client or restricts the client to a sandbox mode. In this mode, the client can only reset the expired password.
```

File: docs/advisors/checks/mysql_password_expiry.md (hunk 3 above)
Comment (rnovikovP) [inline docs/advisors/checks/mysql_password_expiry.md:12]:
```suggestion
This action restores normal access for the session, as well as for subsequent connections that use the account. 
```

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**